src/index.html

```diff
@@ -7,6 +7,70 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="wrapper">
+      <div class="calendar calendar--month-length--31">
```

Comment:
```suggestion
      <div class="calendar calendar--month-length--31 calendar__day--start-day--mon">
```
All modifiers should be used for the calendar class, not the day